package com.revature.exception;

public class UsernameAlreadyTakenException extends Exception {

}
